Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[workspace]
+
+members = ["senior_game_client", "senior_game_server", "senior_game_shared"]
+
+[profile.dev]
+opt-level = 0
```

Comment:
Is opt-level 0 recommended for dev? I know in C++ you use generally still use opt-level 1

Reply:
it appears I mixed up the verdict on a bevy thread, yes that should be opt-level 1